apps/server/src/shared/infra/antivirus/antivirus.service.ts

```diff
@@ -17,20 +17,16 @@ export class AntivirusService {
 	) {}
 
 	public send(requestToken: string | undefined) {
-		try {
-			if (this.options.enabled && requestToken) {
-				const downloadUri = this.getUrl(FilesStorageInternalActions.downloadBySecurityToken, requestToken);
-				const callbackUri = this.getUrl(FilesStorageInternalActions.updateSecurityStatus, requestToken);
+		if (this.options.enabled && requestToken) {
+			const downloadUri = this.getUrl(FilesStorageInternalActions.downloadBySecurityToken, requestToken);
+			const callbackUri = this.getUrl(FilesStorageInternalActions.updateSecurityStatus, requestToken);
 
-				this.amqpConnection.publish(
-					this.options.exchange,
-					this.options.routingKey,
-					{ download_uri: downloadUri, callback_uri: callbackUri },
-					{ persistent: true }
-				);
-			}
-		} catch (err) {
-			throw new InternalServerErrorException(err, AntivirusService.name);
+			this.amqpConnection.publish(
+				this.options.exchange,
+				this.options.routingKey,
+				{ download_uri: downloadUri, callback_uri: callbackUri },
+				{ persistent: true }
+			);
```

Comment:
When we get an error on this place it must be an InternalServerError. Yes it is mapped in the last step to one. But for our domain layer it must have the 500er status code, the error from amqp is maybe interessting but for this cause if enough. 
The core message is something go wrong and you should not handle it, devop / devs must be checked.

This message do not change if we introduce the error mapping.